src/ui/public/react_components.js

```diff
@@ -2,10 +2,15 @@ import 'ngreact';
 
 import {
   KuiToolBarSearchBox,
+  KuiConfirmModal,
 } from 'ui_framework/components';
 
 import { uiModules } from 'ui/modules';
+
 const app = uiModules.get('app/kibana', ['react']);
 app.directive('toolBarSearchBox', function (reactDirective) {
   return reactDirective(KuiToolBarSearchBox);
 });
+app.directive('confirmModal', function (reactDirective) {
```

Comment:
Are there any future plans for removing this and being able to use React components like we would use Angular components? Or, I'm assuming this gets auto-loaded somewhere in the Angular bootup, so is the plan to no longer `import` the components we need wherever we need them? I'm a bit out of touch with our "React in Angular" plans...